Learn/adapters/LearnApi.swift

```diff
@@ -9,16 +9,34 @@
 import Alamofire
 
 struct LearnApi {
+    static func getProfile(completion: @escaping (Student) -> ()) {
+        let headers: HTTPHeaders = [
```

Comment:
This is duplicated from #getCanonicalProgress, but I went the dirty route of just making this work for now.